examples/node-typescript-app/helm/README.md

```diff
@@ -0,0 +1,3 @@
+# my-service Helm Chart
+
+This is a default chart created with `helm create` 
```

Comment:
Since a lot of people (including myself) aren't familiar with Helm, would you mind including a very short description and link to https://helm.sh?